var/spack/repos/builtin/packages/dlb/package.py

```diff
@@ -48,5 +53,7 @@ def configure_args(self):
         args.extend(self.enable_or_disable("debug"))
         args.extend(self.enable_or_disable("instrumentation-debug", variant="debug"))
         args.extend(self.with_or_without("mpi"))
+        args.extend(self.with_or_without("hwloc"))
+        args.extend(self.with_or_without("papi"))
```

Comment:
Doe this work also with older versions, or should we make the option conditional on `spec.satisfies("@3.4:")` ?

Reply:
Ah, yes. The `--with*-papi`  is only available from v3.4 onwards.

```suggestion
        if self.spec.satisfies("@3.4:"):
            args.extend(self.with_or_without("papi"))
```
I think this solves it, let me know otherwise. Thanks!